power-platform/admin/api-request-limits-allocations.md

```diff
@@ -71,13 +71,13 @@ A separate limit is established for certain activities(for example: A background
 - [Application users](create-users.md#create-an-application-user)
 - [Non-interactive users](create-users.md#create-a-non-interactive-user-account)
 - [Administrative users](create-users.md#create-an-administrative-user-account)
-- [SYSTEM user](/dynamics365/customer-engagement/web-api/systemuser?view=dynamics-ce-odata-9#operations)
+- [SYSTEM user](/dynamics365/customer-engagement/web-api/systemuser#operations)
```

Comment:
@PriyaKodukula Should this really be linked to a specific version of the docs? The build system is asking for us to confirm  this. If not, should I link it to this: https://learn.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/systemuser#operations